UnleashClient/periodic_tasks/fetch_and_load.py

```diff
@@ -35,6 +39,15 @@ def fetch_and_load_features(
 
     if state:
         cache.set(FEATURES_URL, state)
+        if event_callback:
+            event = UnleashFetchedEvent(
+                event_type=UnleashEventType.FETCHED,
+                event_id=uuid.uuid4(),
+                raw_features=state,
+            )
+            event_callback(event)
+        if ready_callback:
+            ready_callback()
```

Comment:
I think I kind of like the idea of removing the if statement and having functions doing nothing.